hindsight/ConnectFlow/UI/LoginViewController.swift

```diff
@@ -14,17 +14,37 @@ import RxSwift
 
 class LoginViewController: UIViewController {
 
-    lazy var facebookConnect: UIButton = {
-        let button = UIButton()
-        button.setTitle("Connect With Facebook", for: .normal)
-        button.backgroundColor = ColorName.hindsightBlue.color
-        return button
-    }()
-
     let viewModel: LoginViewModelProtocol
 
     let disposeBag = DisposeBag()
 
+    lazy var logoImageView: UIImageView = {
+        let view = UIImageView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var backgroundImageView: UIImageView = {
+        let view = UIImageView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var facebookConnect: HindsightLoginButton = {
+        let normalState = LoginButtonState(text: "Login With Facebook",
+                                           textColor: ColorName.hindsightWhite.color,
+                                           backgroundColor: ColorName.facebookNormalBlueColor,
+                                           image: UIImage(named: "fb_logo"))
+        let highlightState = LoginButtonState(backgroundColor: ColorName.facebookHighlightBlueColor)
+        let buttonViewModel = LoginButtonViewModel(normalState: normalState,
+                                                   highlightState: highlightState,
+                                                   didSelectClosure: { [unowned self] in
+            self.viewModel.connectFacebook()
```

Comment:
The button behavior should be injected from coordinator. Create the button here induces an hard dependency. Which might be OK for the short term as this is just UI but might need refactor later on.